frontend/src/v5/ui/routes/viewer/theme.ts

```diff
@@ -239,6 +239,22 @@ export const theme = createTheme(
 						},
 					},
 				},
+				MuiChip: {
```

Comment:
The MuiChip styling code (that was in the standard v5 `theme`) has been split between this file (viewer `theme`) and the `tickets.overrides` file, but I don't understand why

Reply:
Yeah, this was silly of me. I should have put the overrides stuff in the chip.styles file (I have amended this now). This was done so that the style of the search filter chips in other cards wouldn't be affected